README.md

```diff
@@ -133,8 +133,8 @@ You can use the following as standard names when invoking `phpcs` to select snif
   - `WordPress-Docs` - additional ruleset for [WordPress inline documentation standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/)
   - `WordPress-Extra` - extended ruleset for recommended best practices, not sufficiently covered in the WordPress core coding standards
     - includes `WordPress-Core`
-  - `WordPress-VIP` - extended ruleset for [WordPress.com VIP coding requirements](https://vip.wordpress.com/documentation/code-review-what-we-look-for/)
-    - includes `WordPress-Core`
+
+**Note:** The `WordPress-VIP` ruleset, which includes `WordPress-Core`, is also currently present as a subset of `WordPress`. It was originally intended for [WordPress.com VIP coding requirements](https://vip.wordpress.com/documentation/code-review-what-we-look-for/), but this is no longer used or recommended by the WordPress.com VIP team or their clients. **This ruleset is considered to be deprecated, and will be removed in due course.** See [#1309](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1309) for more information.
```

Comment:
Would this work?

> **Note:** The `WordPress-VIP` ruleset, was originally intended for [WordPress.com VIP coding requirements](https://vip.wordpress.com/documentation/code-review-what-we-look-for/), but this is no longer used or recommended by the WordPress.com VIP team or their clients. **This ruleset is deprecated, and will be removed in due course.** See [#1309](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1309) for more information. Instead, for working with VIP projects consider using [the official VIP coding standards](https://github.com/Automattic/VIP-Coding-Standards).

Reply:
I like the historic perspective in your new text snippet, though the text doesn't really flow right yet.
I would also like to see it mentioned that the VIP ruleset used to be included in the `WordPress` ruleset, but that as the ruleset has been deprecated as of WPCS 1.0.0, it no longer is.

Reply:
> This WPCS package contains the sniffs for another ruleset, `WordPress-VIP`. This ruleset was originally intended to aid with the [WordPress.com VIP coding requirements](https://vip.wordpress.com/documentation/code-review-what-we-look-for/), but this is no longer used or recommended by the WordPress.com VIP team or their clients, since they prefer to use their [official VIP coding standards](https://github.com/Automattic/VIP-Coding-Standards) ruleset instead.
>
> Before WPCS `1.0.0`, the `WordPress-VIP` ruleset was included as part of the complete `WordPress` ruleset. **As of `1.0.0` the `WordPress-VIP` ruleset is not part of the `WordPress` ruleset, and it is deprecated**. The remaining `WordPress-VIP` sniffs may still be referenced in custom rulesets, so to maintain some backwards compatibility, they will remain in WPCS until `2.0.0`. See #1309 for more information.

Reply:
@jrfnl  I decided to approach this with a fresh batch of coffee and came up with this:

> **Note:** The `WordPress-Core` ruleset used to include the `Wordpress-VIP` ruleset, but the VIP ruleset has been deprecated, and will be removed in a future release. The `WordPress-VIP` ruleset includes the `WordPress-Core` ruleset. The VIP ruleset was originally intended for working with [WordPress.com VIP sites](https://vip.wordpress.com/documentation/code-review-what-we-look-for/), but has been replaced by the [VIP Coding Standards](https://github.com/Automattic/VIP-Coding-Standards), and is no longer used or recommended by the WordPress.com VIP team or their clients. **This ruleset is considered to be deprecated, and will be removed in due course.** See [#1309](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1309) for more information.

Reply:
@tomjn Looks good, just some minor points:
1. `WordPress-VIP` was never included in `WordPress-Core`. It was included in the overall `WordPress` ruleset.
2. I think it could be helpful to mention that the deprecation - and the removal of VIP from the `WordPress` ruleset - happened in v 1.0.0.
3. I think this bit can be left out as, while still true, it's not really relevant anymore: `The WordPress-VIP ruleset includes the WordPress-Core ruleset. `
4. `no longer used or recommended by the WordPress.com VIP team` + "nor the WPCS team".